runtime/src/accounts.rs

```diff
@@ -738,8 +739,7 @@ impl Accounts {
                 );
                 if message.is_writable(i) {
                     if account.rent_epoch == 0 {
```

Comment:
Well, these inconsistent predicates are concerning me. Ideally, I want to put everything into `RentCollector::update`/`RentCollector::from_existing_account()`. But, to avoid any additional risks, I've punted it. ;)